assets/elasticsearch/Dockerfile

```
@@ -1,21 +1,24 @@
-FROM ubuntu:latest
+FROM ghcr.io/y-scope/clp/clp-core-dependencies-x86-ubuntu-jammy:main
 
 RUN apt-get update \
     && DEBIAN_FRONTEND=noninteractive apt-get install -y \
     curl \
-    gpg
+    gpg;
 
-RUN curl -fsSL https://artifacts.elastic.co/GPG-KEY-elasticsearch | gpg --dearmor -o /usr/share/keyrings/elasticsearch-keyring.gpg \
-    && echo "deb [signed-by=/usr/share/keyrings/elasticsearch-keyring.gpg] https://artifacts.elastic.co/packages/8.x/apt stable main" | tee /etc/apt/sources.list.d/elastic-8.x.list
+RUN curl -fsSL https://artifacts.elastic.co/GPG-KEY-elasticsearch \
```

Hadolint (2.12.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)